src/chainstate/burn/operations/leader_block_commit.rs

```diff
@@ -598,14 +598,14 @@ impl LeaderBlockCommitOp {
                             error!("Failed to check whether parent (height={}) is descendent of anchor block={}: {}",
                                    parent_block_height, &reward_set_info.anchor_block, e);
                             op_error::BlockCommitAnchorCheck})?;
-                    if descended_from_anchor != expect_pox_descendant {
-                        if descended_from_anchor {
-                            warn!("Invalid block commit: descended from PoX anchor, but used burn outputs");
-                        } else {
-                            warn!(
+                    if descended_from_anchor && !expect_pox_descendant {
```

Comment:
THIS IS POTENTIALLY CONSENSUS BREAKING.  Please undo this!

Reply:
Removed.